docs/build.gradle

```diff
@@ -40,9 +40,12 @@ cppProjectZips.add(project(':wpinet').cppHeadersZip)
 cppProjectZips.add(project(':wpiutil').cppHeadersZip)
 
 doxygen {
-    executables {
-        doxygen version : '1.9.4',
-        baseURI : 'https://frcmaven.wpi.edu/artifactory/generic-release-mirror/doxygen'
+    // Doxygen binaries are unavailable for arm64, they must be provided by the system
+    if (!it.targetPlatform.operatingSystem.isArm64()) {
```

Comment:
Is Arm64 the only platform this applies to?
Maybe check on the positive that the current platform is one this works for?

Reply:
Changed it to positive - the binaries are only provided for intel 64-bit platforms.

Reply:
It still tries to use arm binaries as is. That part of the check should be removed.

Reply:
> That part of the check should be removed if only intel works

Sorry, the check is (after changes to original comment) currently for x86-64 or AMD64, that would preclude arm platforms right?